index.js

```diff
@@ -69,7 +69,7 @@ app.get('/api/airports', async function(req, res) {
     qs = "SELECT airportname from `travel-sample` WHERE LOWER(airportname) LIKE '%" + searchTerm.toLowerCase() + "%';";
   }
 
-  let result = await coll.query(qs);
+  let result = await cluster.query(qs).catch(err => console.log(err));
```

Comment:
You're mixing async/await with normal promise usage here.  You should be able to just wrap this await call in a try-catch block rather than using the catch method.